### ISSUE TITLE
Support for un-deduplication?

### ISSUE THREAD

**Issue by petsva** [nlnwa/warchaeology#172]
Have you considered a possibility to re-duplicate/un-deduplicate a warc file (or a set of warc files) in some way? Or is that already possible?

For example if you want to export all data in a set of  warc files from a larger set, plus all data they reference through revisit records. Either through replacing revisit records with actual records or through also exporting the revisit-referenced files, but let them only contain the records that were referenced.

**Comment by maeb** [comment]
Yes, we would like to implement reduplication/unduplication. It is not possible now.

An implementation would replace one of the revisit records with a response (or other type of referenced record) and replace/modify other revisit records to reference this new record. I think it would also be useful to be able to fully unduplicate a dataset by replacing all revisit records.

**Comment by petsva** [comment]
Nice!

When you export parts of WARC files it is maybe not necessary to replace revisit records "in place" to response records. You could instead extract all visit record referenced responses to a new "revisit target" WARC, and just update the references in the revisit records to this new WARC.

The scenario I am thinking of is to extract all data harvested from a certain seed in a set of WARC files.
